src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -111,7 +118,12 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
   }
 
   async componentDidMount() {
-    const { store, dependentStores, isClusterScoped } = this.props;
+    const { store, dependentStores, isClusterScoped, isConfigurable, tableId } = this.props;
+    if (isConfigurable && tableId) {
+      const filters = get(userStore.preferences.tablesConfiguration, tableId);
```

Comment:
We don't seem to use `get` and `set` functions from `mobx` in this repo. Please change to plain accesses.
```suggestion
      const filters = userStore.preferences.tablesConfiguration.get(tableId);
```